### ISSUE TITLE
Version issue (v3rc01 -> v3rc02)

### ISSUE THREAD

**Issue by Dorocy** [FraunhoferIOSB/FAAAST-Service#523]
hello.
I have an Version issue with Faaast.
Faaast supports the details of Asset Administration Shell V3RC01 (PART1). 
recently, Package Explorer has been updated to v3 alpha version, which now supports the details of Asset Administration Shell V3RC02 (PART1).
As a result, the json result file generated by Package Explorer(v3 alpha version) is not compatible with Faaast due to the version mismatch.
Additionally, the result file can't be converted using 'Faaast-package-explorer-Converter'.

I would like to know Faaast has any plans to update to V3RC02, and if so, when we can expect the update. I'm looking forward to the new version of Faaast. 

Thank you.

**Comment by mjacoby** [comment]
Unfortunately, things are a little bit more complex than this.
FA³ST Service uses a combination of v3.0RC01 and v3.0RC02 for the metamodel but is not 100% compliant with either one.

Actually, FA³ST Service uses https://github.com/admin-shell-io/java-model/ as underlying library which I am also one of the developers. It started with the plan to implement v3.0RC01 but discovered some issues with the specification that made proper implementation impossible. As a result, this library is neither compliant to v3.0RC01 nor v3.0RC02 although some of the fixes made during implementation were later included in v3.0RC02.

AASX Package Explorer has also not been 100% compliant with v3.0RC01 so far. Because of this unsatisfying state, we developed https://github.com/FraunhoferIOSB/FAAAST-Package-Explorer-Converter. However, it was only a rudamentary tool working for exact that combination of FA³ST Service and AASX Package Explorer. It was never intended to be used indefinitely but to become obsolete once both tools implement the final v3.0 of the metamodel. 

Current state is, that v3.0 of the specification has been released some months ago. The library FA³ST Service uses for the metamodel has now become an Eclipse project https://github.com/eclipse-aas4j/aas4j. However, this library has not yet a stable release for v3.0 but only a beta/milestone release. 
We are actively working on updating FA³ST Service to that release of AAS4j and you can find the current status here https://github.com/FraunhoferIOSB/FAAAST-Service/tree/feature/metamodel-v3.0. Please be aware that this is a working development branch and therefore might change behavior or break at any moment.

In conclusion, FA³ST Service will never support v3.0RC02 but move directly to final v3.0, probably within the next months.
We are also not planning on putting additional work into the FA³ST Package Explorer Converter as it is expected to become obsolete once everybody updates to v3.0. 

I therefore see basically two solutions to your problem - either keep using the older version of AASX Package Explorer that is compatible with FA³ST or try to update FA³ST Package Explorer Converter yourself. If you should decide for the second options and want to share the code with the world feel free to create a Pull Request.
